docs/assets/site.js

```
@@ -0,0 +1,168 @@
+/* global anchors */
+
+// add anchor links to headers
+anchors.options.placement = 'left';
+anchors.add('h3');
+
+// Filter UI
+var tocElements = document.getElementById('toc').getElementsByTagName('li');
+
+document.getElementById('filter-input').addEventListener('keyup', function (e) {
+  var i, element, children;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)